src/js/components/ProgressBar.js

```diff
@@ -79,22 +79,22 @@ class ProgressBar extends React.Component {
     stealPortion(barSizes, indexesLessThanThreshold, unassignedPortion);
 
     return data.map(function(status, index) {
-      let { className, style = {} } = status;
+      let { state, style = {} } = status;
```

Comment:
I never like shadowing variables, and seems especially nasty when it's called `state` in a react component. I'd suggest a rename.

Reply:
❣️ I do suggest `statusClassName`

Reply:
@MatApple and I had decided to move to `state` instead of `className`.

I'm indifferent changing to `statusClassName`. Please let me know the verdict.